src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitDaoImplTest {
+    private FruitDao fruitDao;
+
+    @BeforeEach
+    void setUp() {
+        fruitDao = new FruitDaoImpl();
```

Comment:
better initialize FruitDao only once in the method BeforeAll (check other test classes)